src/app/item-page/simple/item-types/shared/item.component.ts

```diff
@@ -38,12 +45,33 @@ export class ItemComponent implements OnInit {
 
   mediaViewer;
 
-  constructor(protected routeService: RouteService) {
+  constructor(protected routeService: RouteService,
+              protected router: Router) {
     this.mediaViewer = environment.mediaViewer;
   }
 
+  /**
+   * The function used to return to list from the item.
+   */
+  back = () => {
+    this.routeService.getPreviousUrl().pipe(
+          take(1)
+        ).subscribe(
+          (url => {
+            this.router.navigateByUrl(url);
+          })
+        );
+  };
+
   ngOnInit(): void {
+
     this.itemPageRoute = getItemPageRoute(this.object);
+    // hide/show the back button
+    this.showBackButton = this.routeService.getPreviousUrl().pipe(
+      filter(url => /^(\/search|\/browse|\/collections|\/admin\/search|\/mydspace)/.test(url)),
```

Comment:
We might think about turning this RegEx into a constant at the top of this component (to make it easier to find).  I suspect it will need to change for some (highly custom) DSpace sites.. or even in the future for DSpace itself.  While I don't think it needs to be made configurable yet, it could be nice to simply make it easier to find/update at the component level.

Reply:
That's a good suggestion, @tdonohue . It's done.